ethcore/sync/src/light_sync/mod.rs

```diff
@@ -699,6 +725,10 @@ pub trait SyncInfo {
 
 	/// Whether major sync is underway.
 	fn is_major_importing(&self) -> bool;
```

Comment:
IS this still used?

Reply:
yes for light clients

Reply:
Can't light clients use the same mechanism instead? it seems that it triggers notifications as well, no?